conda_requirements.txt

```diff
@@ -1,7 +1,7 @@
 numpy>=1.16
 dask>=2
 dask[complete]>=2
-xarray>=0.15
+xarray>=0.19
```

Comment:
Do we want to consider pinning the version? It's possible to restrict the version to only "minor" changes like bugfixes. i.e.:
```
xarray~=0.19.0
```
Will mean we can get 0.19.1, 0.19.2, etc. but not 0.20 and beyond, until we decide to move to the new major version.